backend/lib/site.py

```diff
@@ -8,3 +8,30 @@ def build_full_url(url):
     domain_name = Site.objects.first().domain
     normalized_url = re.sub(r'^/', '', url)
     return '{}{}/{}'.format(protocol, domain_name, normalized_url)
+
+
+def _get_this_site_url() -> str:
```

Comment:
I think [that method](https://github.com/TheSpaghettiDetective/obico-server/pull/828/files#diff-d05647a89b73f00aa43c2a26ee9c60809918d2dcf575737fc34567318c0db84aR6) can be DRY'd a little more by calling this method.

Reply:
fixed in [f4678db](https://github.com/TheSpaghettiDetective/obico-server/pull/828/commits/f4678db6a21fc07b5370621dd4b3376256a9c465)

Reply:
I was thinking about the other way around. But your change is probably more elegant.